### DISCUSSION TITLE
Wix toolset v4 multilanguage

### DISCUSSION THREAD

**Discussion by vanchor** [wixtoolset/issues#7628] [Questions]
Hi, how can i create multilanguage bundle installer in wix toolset v4? I have BootstrapperApplication with LocalizationFile for 1033 code and custom themeFile, but i also need to add another languages that will be chosen for user region (or it also can be some ui with list of languages)

I tried to add another wxl files for different language code, but on build i have error like "The localization identifier 'Caption' has been duplicated in multiple locations. Please resolve the conflict."

**Comment by cz-dev-ge** [comment]
See [Discussion 9081](https://github.com/orgs/wixtoolset/discussions/9081) for a workaround.

Short steps (given your language file is named `RtfTheme.wxl`):

+ duplicate `RtfTheme.wxl` as `RtfTheme.en-US.wxl`
+ remove `Culture="en-us"` from `RtfTheme.wxl`
+ add `<Payload Name="1033\thm.wxl" SourceFile="RtfTheme.en-US.wxl" />`

More details in the linked discussion.
